### PR TITLE
Update Run.bat

### PR DESCRIPTION
**Pull request by MazeEQ**
1. Added launching of D2r Assist from batch
2. Fixed grammatical errors
3. Formatting consistency

### CODE REVIEW

**Comment by misterokaygo** [comment]
We are close just a couple more things

File: D2MapApi/Run.bat [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+reg query "HKCU\SOFTWARE\Blizzard Entertainment\Diablo II" /v "InstallPath" >nul 2>nul
+if %ERRORLEVEL% EQU 1 goto GetD2Path
+	:AutoPath
+		for /F "tokens=2*" %%a in ('reg query "HKCU\SOFTWARE\Blizzard Entertainment\Diablo II" /v "InstallPath"') do (set InstallPath=%%b)
+			if "%InstallPath:~-1%"=="\" set InstallPath=%InstallPath:~0,-1%
+				goto Launch
+	:GetD2Path
+		echo Diablo II 1.13c directory was not detected
+			set /P InstallPath="Please enter the location of your Diablo 2 1.13c directory: 
+				for %%G in ("%InstallPath%") do if "%%~aG" Lss "d" if "%%~aG" GEq "-" (
```
Comment (misterokaygo) [inline D2MapApi/Run.bat:11]: is `Lss` case-sensitive?

**Review by misterokaygo — CHANGES_REQUESTED** [review]

File: D2MapApi/Run.bat [hunk 2]
```diff
@@ -1,2 +1,17 @@
...
+					goto GetD2Path ) else goto GetD2Path
+						reg add "HKCU\SOFTWARE\Blizzard Entertainment\Diablo II" /v "InstallPath" /t REG_SZ /d "%InstallPath%" /f
+							goto :Launch
+	:Launch
+		start cmd /k d2mapapi "%InstallPath%"
+			for /f %%G in ('dir /b %~dp0..\*.exe') do ( start %~dp0..\%%G)
```
[the remark this answers is not in the document]
Comment (MazeEQ) [reply D2MapApi/Run.bat:17]: The space is required.

**Comment by AudriusButkevicius** [comment]
This PR should also target develop branch I think, rather than main. If you are not very git proficient, might be easier to start from scratch making changes on that branch.

**Review by OneXDeveloper — CHANGES_REQUESTED** [review]
Please remove additional changes

**Review by AudriusButkevicius — APPROVED** [review]

**Comment by AudriusButkevicius** [comment]
Actually, I'll just port your changes to a new PR.

**Pull request closed without merging**